Error with Apple Silicon

**Description of the bug:**

CloudNet produces the following error when running on a processor with the new Apple Silicon architecture:
```
Starting CloudNet...
java.lang.reflect.InvocationTargetException
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at de.dytanic.cloudnet.launcher.CloudNetLauncher.lambda$startApplication$4(CloudNetLauncher.java:297)
at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.UnsatisfiedLinkError: Could not load library. Reasons: [no jansi in java.library.path: [/usr/java/packages/lib, /lib, /usr/lib], /tmp/libjansi-64-5645445519440994379.so: /tmp/libjansi-64-5645445519440994379.so: cannot open shared object file: No such file or directory (Possible cause: can't load AMD 64-bit .so on a AARCH64-bit platform)]
at org.fusesource.hawtjni.runtime.Library.doLoad(Library.java:182)
at org.fusesource.hawtjni.runtime.Library.load(Library.java:140)
at org.fusesource.jansi.internal.CLibrary.<clinit>(CLibrary.java:42)
at org.fusesource.jansi.AnsiConsole.wrapOutputStream(AnsiConsole.java:48)
at org.fusesource.jansi.AnsiConsole.<clinit>(AnsiConsole.java:38)
at de.dytanic.cloudnet.console.JLine2Console.<init>(JLine2Console.java:45)
at de.dytanic.cloudnet.Main.main(Main.java:24)
... 6 more
```

**Steps to reproduce this bug:**

Start CloudNet on a computer with the new processor.

**Expected behavior:**

CloudNet should start without any exceptions

--- a/cloudnet/src/main/java/de/dytanic/cloudnet/console/JLine3Console.java
+++ b/cloudnet/src/main/java/de/dytanic/cloudnet/console/JLine3Console.java
@@ -50,7 +50,7 @@ public final class JLine3Console implements IConsole {
 
         try {
             AnsiConsole.systemInstall();
-        }catch (UnsatisfiedLinkError ignored) {
+        } catch (UnsatisfiedLinkError ignored) {
         }
 
         this.terminal = TerminalBuilder.builder().system(true).encoding(StandardCharsets.UTF_8).build();

--- a/cloudnet/src/main/java/de/dytanic/cloudnet/console/JLine3Console.java
+++ b/cloudnet/src/main/java/de/dytanic/cloudnet/console/JLine3Console.java
@@ -48,8 +48,9 @@ public final class JLine3Console implements IConsole {
     public JLine3Console() throws Exception {
         System.setProperty("library.jansi.version", "CloudNET");
 
-        if (!System.getProperty("os.version").contains("raspi")) {
+        try {
             AnsiConsole.systemInstall();
+        }catch (UnsatisfiedLinkError ignored) {
         }
 
         this.terminal = TerminalBuilder.builder().system(true).encoding(StandardCharsets.UTF_8).build();

--- a/cloudnet/src/main/java/de/dytanic/cloudnet/console/JLine3Console.java
+++ b/cloudnet/src/main/java/de/dytanic/cloudnet/console/JLine3Console.java
@@ -50,7 +50,7 @@ public final class JLine3Console implements IConsole {
 
         try {
             AnsiConsole.systemInstall();
-        } catch (UnsatisfiedLinkError ignored) {
+        } catch (Throwable ignored) {
         }
 
         this.terminal = TerminalBuilder.builder().system(true).encoding(StandardCharsets.UTF_8).build();

